java/java.source.base/test/unit/src/org/netbeans/modules/java/source/save/FormatingTest.java

```diff
@@ -5201,6 +5201,51 @@ public void testJavadoc() throws Exception {
                 + "}\n";
         reformat(doc, content, golden);
 
+        
+        content ="package hierbas.del.litoral;\n" +
+                    "\n" +
```

Comment:
feel free to bump the java version to 17 to unlock text blocks.
replace
https://github.com/apache/netbeans/blob/08339224a1417114ef2838fccbc5fcfeeb992c1e/java/java.source.base/nbproject/project.properties#L21
with `java.release=17`